detect_test.go

```diff
@@ -91,8 +91,9 @@ api = "0.5"
 				context = ctx
 
 				return packit.DetectResult{}, nil
-			}, packit.WithArgs([]string{binaryPath, platformDir, planPath}))
+			}, packit.WithArgs([]string{binaryPath, platformDir, planPath}), packit.WithExitHandler(exitHandler))
```

Comment:
I feel like we should add a test for the extension.toml code path as well for completeness